README.md

```diff
@@ -664,4 +664,21 @@ When running in insecure mode, the secrets are retrieved from the *Writable.Inse
 
 #### Storing Secrets
 
-<placeholder>
\ No newline at end of file
+Secrets for an application service can be store in the secret store (Vault) by making an HTTP POST to '/secrets'.  An example of the message body JSON is below.  Once a secret is stored only the service that added the secret will be able to retrieve it.  For secret retrieval see [Getting Secrets](#getting-secrets) above.
```

Comment:
1. change: can be store to can be **stored**
2. I believe the HTTP Post is not just to /secrets but more specifically to the concatenated string of path (from the config) + subpath (from the API)
3. nitpick: add comma - "Once a secret is stored,"